src/main/scala/com/fulcrumgenomics/illumina/SampleSheet.scala

```diff
@@ -100,14 +100,25 @@ object SampleSheet {
 
     // get the rows (values), so skip "[Data]" and the header
     val lineNumber = preData.size + 2 // 0-based
-    val sampleDatums = postData.drop(2).zipWithIndex.map { case (line, rowNumber) =>
+    val maybeSampleDatums = postData.drop(2).zipWithIndex.map { case (line, rowNumber) =>
         val values = line.split(SplitRegex, -1)
+        // treat as empty line iff all columns are empty (empty line or comma-only line, often added by spreadsheet editors)
+        if (values.forall(_.trim.isEmpty)) None
```

Comment:
This will allow rows where the # of fields is wrong _in the middle_ of the data section, not just the end.

Reply:
It sets the row `datum` to `None`, which is then checked below when creating `sampleDatums`. 

If there are any `None`s mixed in the middle of the data section it throws the error:

```
"Found empty rows in sample data section: ${emptyRows.mkString(", ")}"
```